### ISSUE TITLE
1.0.8 error

### ISSUE THREAD

**Issue by ljuwon321** [Jire/Charlatano#601]
not working skeleton, box esp.
and cheat only working on Full Screen. not working others.
and not working bomb timer, esp.

if i use full screen, glow esp is working. but not only black screen on game window.
i can't see anything.

my pc windows is
Windows 10 Home 1803 version
i don't install any anti-virus program. and i turn off my window defender.

Somebody fix or give solution plz.

**Comment by Baffy** [comment]
yeah but if you tap with pistol or any other weapon the rcs is pulling down 
and when shooting its really obvious you cheating because how the image is moving 😣

**Comment by Primeknight19** [comment]
Same thing is happening with me, while shooting my aim goes down(specially using AUG or SSG rifle) but somehow i get the kill. Please fix this

**Comment by Atlanticore** [comment]
RCS is a little broken right now

**Comment by Baffy** [comment]
well for me the esp works.. 
I have disabled aim and rcs and tryed playing whithout them.
works but its not the same thing as before 😞

**Comment by Primeknight19** [comment]
You can turn aim and rcs ON. actually the bug is helpful. If someone is watching us play. they wont notice that we are using hack. so the bug is okay, its actually helpful. not creating a major problem

**Comment by Baffy** [comment]
really? ... try one tapping with a pistol and see how good it helps 🤣

**Comment by ljuwon321** [comment]
glow esp is working.
but skeleton and box esp not working. ok?
